charts/akeyless-api-gateway/values.yaml

```diff
@@ -294,10 +294,14 @@ TLSConf:
   enableSniProxy: false
   # minimumTlsVersion can be one of the following <TLSv1/TLSv1.1/TLSv1.2/TLSv1.3> 
   minimumTlsVersion:
-  # Specifies an existing secret for tls-certificate must include:
-  # - akeyless-api-cert.crt (base64)
```

Comment:
tlsExistingSecretName is looking for an existing k8s secret with these field and structure - otherwise, it will not work. so it need to be mentioned , also you need to check for backwards compatibility to support both old and new deployment ways without breaking or changing his values file

Reply:
Hey @niv-ez these lines copy the input file into the output file that is actually used by the gateway https://github.com/akeylesslabs/helm-charts/blob/existing-secret-input-file-override/charts/akeyless-api-gateway/templates/deployment.yaml#L100-L105